terraform/modules/benchmark/service/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:22.04
+
+# Prevent interactive prompts during package installation
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install necessary forest and lotus packages
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+    ruby ruby-dev make gcc git git-lfs curl wget pkg-config clang build-essential \
+    mesa-opencl-icd ocl-icd-opencl-dev hwloc libhwloc-dev s3cmd aria2 zstd time bzr jq pkg-config && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I'm a bit puzzled. Why do we need all of those dependencies? Could we just use Docker? We want to use the latest releases of both Forest and Lotus, which are conveniently dockerized. This would reduce the amount of manual work in the benchmark itself.

That said, I'm okay with doing this in a separate issue. If so, let's create it.

Reply:
I'll create a separate issue for this